protocol/writing-resource/post-target-not-found.feature

```diff
@@ -0,0 +1,43 @@
+Feature: POST to non-existing resource must result in 404
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.reserveContainer()  
+    * def rdfResource = testContainer.reserveResource('.ttl');
+    * def jsonResource = testContainer.reserveResource('.json');
+    * def fooResource = testContainer.reserveResource('.foo');
+    # TODO: Is it better to reserve these in the scenarios?
```

Comment:
Either works but given that they are not re-used I would tend to put them within the scenarios as this avoids creating redundant objects when the Background is called for each Scenario.

Reply:
Right, I'll do that.